src/Symfony/Component/AssetMapper/Event/PreAssetsCompileEvent.php

```diff
@@ -21,20 +21,13 @@
  */
 class PreAssetsCompileEvent extends Event
 {
-    private string $outputDir;
     private OutputInterface $output;
 
-    public function __construct(string $outputDir, OutputInterface $output)
+    public function __construct(OutputInterface $output)
     {
-        $this->outputDir = $outputDir;
         $this->output = $output;
     }
 
-    public function getOutputDir(): string
-    {
-        return $this->outputDir;
-    }
```

Comment:
I created this event - and just included `$outputDir` because I had it available. Now that it's not readily available, I don't think it's a big deal to not include it. My use-case for this event is to run some code that pre-compiles some internal files (not into the output directory) so that when the true compilation process happens, those files are ready and available.